src/workerd/io/hibernation-manager.c++

```diff
@@ -207,7 +207,7 @@ kj::Promise<void> HibernationManagerImpl::readLoop(HibernatableWebSocket& hib) {
               // autoResponseTimestamp on the active websocket.
               (active)->setAutoResponseTimestamp(hib.autoResponseTimestamp);
             }
-            ws.send((reqResp)->getResponse().asArray());
+            co_await ws.send((reqResp)->getResponse().asArray());
```

Comment:
@jqmmes what would you like to do here? Have it `co_await` or continue discarding until your fix is in?

Reply:
It's OK to merge it as is. I can update it later, once my PR is ready.